src/Faker/Provider/es_MX/Person.php

```diff
@@ -0,0 +1,337 @@
+<?php
+
+namespace Faker\Provider\es_MX;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastNameFather}} {{lastNameMother}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastNameFather}} {{lastNameMother}}',
+    );
+
+    protected static $stateAbbr = array(
+        'AS', 'BC', 'BS', 'CC', 'CL', 'CM', 'CS', 'CH', 'DF', 'DG', 'GT', 'GR', 'HG', 'JC', 'MC', 'MN', 'MS', 'NT', 'NL', 'OC', 'PL', 'QT', 'QR', 'SP', 'SL', 'SR', 'TC', 'TS', 'TL', 'VZ', 'YN', 'ZS'
+    );
+
+    /**
+     * {@link} http://www.studentsoftheworld.info/penpals/stats.php3?Pays=MEX
+     */
+    protected static $firstNameMale = array('Carlos', 'Daniel', 'Jose', 'Eduardo', 'Luis', 'Jorge', 'Angel', 'Alejandro', 'Fernando', 'Sergio', 'Ivan', 'Jesus', 'Isaac', 'Cesar', 'Christian', 'Juan Carlos', 'Juan', 'Oscar', 'Adrian', 'Arturo', 'David', 'Diego', 'Enrique', 'Rafael', 'Emmanuel', 'Alex', 'Armando', 'Antonio', 'Rodrigo', 'Erick', 'Alberto', 'José Luis', 'Roberto', 'Andrès', 'Pablo', 'Jonathan', 'Alexis', 'Julio', 'Kevin', 'Edgar', 'Ramiro', 'Miguel', 'Martin', 'Francisco', 'Hugo', 'Ricardo', 'Victor', 'Ruben', 'Humberto', 'MARIO', 'Luis Fernando', 'Gabriel', 'Mauricio', 'Joaquín', 'Israel', 'Heriberto', 'Mariana', 'Rene', 'Raul', 'Hector', 'Hibram', 'Bryan', 'Horacio', 'Benjamin', 'Fede', 'Alan', 'Moises', 'Pepe', 'Fidel', 'Frank', 'Marco antonio', 'Chris', 'Michael', 'Omar', 'Saul Omar', 'Sebastian', 'Aldair', 'Diana', 'Gerardo', 'Eliseo', 'Brian', 'Alexander', 'Roger', 'Juan Manuel', 'Brandon', 'Yoet', 'Abraham', 'Cristian', 'Javier', 'Lalo', 'Marco', 'Maximiliano', 'Salvador', 'Fredy', 'Claudio', 'Esteban', 'Memo', 'Andre', 'Rivera', 'Josue');
+
+    protected static $firstNameFemale = array('Diana', 'Daniela', 'Fernanda', 'Andrea', 'Ana', 'María', 'Alejandra', 'Mariana', 'Karla', 'Gabriela', 'Adriana', 'Jazmin', 'Samantha', 'Itzel', 'Rosa', 'Elizabeth', 'Viridiana', 'Brenda', 'Melissa', 'Sara', 'Jessica', 'Gaby', 'Miriam', 'Karen', 'Esmeralda', 'Laura', 'Frida', 'Alondra', 'Rocio', 'Karina', 'Paola', 'Marisol', 'Ali', 'Dulce', 'Valeria', 'Flor', 'Marian', 'Alexa', 'Tania', 'Paula', 'Michelle', 'Alma', 'Carolina', 'Claudia', 'Yazmin', 'Paulina', 'Cynthia', 'Liz', 'Lorna', 'Erika', 'Denisse', 'Clarisa', 'Perla', 'Natalia', 'Lizeth', 'Rebeca', 'Anahí', 'Fanny', 'LETICIA', 'Mayra', 'Maribel', 'Maria Fernanda', 'Marcela', 'Julia', 'Cecilia', 'Giselle', 'Violeta', 'Nelly', 'Sandy', 'Estefania', 'Abril', 'Cris', 'Silvia', 'Sofia', 'Lupita', 'Beatriz', 'Areli', 'Aimee', 'Victoria', 'Susy Benvidez', 'Noemi', 'Aranza', 'Cristina', 'Naomi', 'LeTy', 'Yesenia', 'Gabby', 'Eunice', 'Jenny', 'Leonor', 'Clarissa', 'Vale', 'Pasett', 'Sarai', 'Jhoaly', 'Annie', 'Angie', 'Dafne', 'Tonantzin', 'Lucy');
+
+    /**
+     * {@link} http://surnames.behindthename.com/submit/names/usage/mexican
```

Comment:
Oh no we can't. The idea isn't to have a valid link, the idea is to have a valid content.

I prefer that you look for a Mexican source.